addons/website/views/snippets/s_freegrid.xml

```diff
@@ -0,0 +1,40 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+
+<template id="s_freegrid" name="Free grid">
+    <section class="s_freegrid o_cc o_cc1 pt64 pb64">
+        <div class="container-fluid">
+            <div class="row o_grid_mode" data-row-count="14" style="gap: 16px;">
```

Comment:
I suggest to remove `gap: 16px;` and use `--grid-item-padding-y: 8px; --grid-item-padding-x: 8px;` on the grid items instead. It will make the snippet fit inside the viewport for most of screen sizes and feels a bit more balanced

| gaps | padding |
|--------|--------|
| <img width="1701" alt="Screenshot 2024-08-22 at 17 07 09" src="https://github.com/user-attachments/assets/92c86b9b-3a18-4c3e-ab1f-fe9dd8299b9b"> | <img width="1702" alt="Screenshot 2024-08-22 at 17 10 50" src="https://github.com/user-attachments/assets/10ba7abb-ac39-43a1-8408-75f0fe39064c"> |

Reply:
Good point. So I've added some inline style padding depending on their position (0 0 for those on the edges, and 8 or 16 for the rest), I also fine-tuned the padding on the text and buttons to ensure everything aligns perfectly on mobile.